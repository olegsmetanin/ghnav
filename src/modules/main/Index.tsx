import { Hello } from 'common/components/Hello'
import React from 'react'

export const Index = () => <Hello />
