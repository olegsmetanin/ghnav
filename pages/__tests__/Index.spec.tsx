/* eslint-env jest */
import Index from '../index'
import React from 'react'
import renderer from 'react-test-renderer'

describe('Index', () => {
  it('renders correct', () => {
    const component = renderer.create(<Index />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
