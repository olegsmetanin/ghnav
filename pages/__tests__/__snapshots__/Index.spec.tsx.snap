// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders correct 1`] = `
<div>
  Hello from nextjs and typescript with love!
</div>
`;
